### DISCUSSION TITLE
Installation on Fedora 35 using RPM

### DISCUSSION THREAD

**Discussion by TekkaTim** [netbox-community/netbox#8434] [Q&A]
Hi Everyone,

I have been recommended Netbox by a college and when I went to install it on Fedora 35, I noticed that there is an RPM for it in the fedora repo.

```
dnf info netbox

Last metadata expiration check: 2:52:46 ago on Sat 22 Jan 2022 12:33:35 PM AEDT.
Installed Packages
Name         : netbox
Version      : 2.11.10
Release      : 1.fc35
Architecture : noarch
Size         : 18 M
Source       : netbox-2.11.10-1.fc35.src.rpm
Repository   : @System
From repo    : fedora
Summary      : IP address management (IPAM) and data center infrastructure
             : management (DCIM)
URL          : https://github.com/netbox-community/netbox/
License      : ASL 2.0 and MIT and OFL
Description  : NetBox is an IP address management (IPAM)
             : and data center infrastructure management (DCIM) tool.
             : Initially conceived by the network engineering team at
             : DigitalOcean, NetBox was developed specifically to address the
             : needs of network and infrastructure engineers. It is intended to
             : function as a domain-specific source of truth for network
             : operations.
```

Has anyone installed it using this rpm? 
I have followed the instructions on [readthedocs](https://netbox.readthedocs.io/en/stable/installation/) up to the point that I need to run the `upgrade.sh` script ("Run the Upgrade Script" section) and there is no such script. I know the script creates a python venv and I could download the script and run it, but was wondering if the RPM has done something about it already (as some of the other install parts are already done, like user creation).

FYI, the RPM has over 3400 files associated with it, and the following is a list of the files that are not in `/usr/share/`.

```
# rpm -ql netbox | wc -l
3443

# rpm -ql netbox | grep -v "usr/share"
/etc/netbox
/etc/netbox/config
/etc/netbox/config/configuration.py
/etc/netbox/config/ldap
/etc/netbox/config/ldap/ldap_config.py
/etc/netbox/gunicorn_config.py
/etc/netbox/reports
/etc/netbox/scripts
/usr/lib/systemd/system/netbox-rq.service
/usr/lib/systemd/system/netbox.service
/var/lib/netbox
/var/lib/netbox/media
/var/lib/netbox/media/devicetype-images
/var/lib/netbox/media/image-attachments
/var/lib/netbox/static
```

Thanks
Tim

**Comment by ziggekatten** [comment]
2.11 is a very old version, and would not recommend going that route.  

Given the pace of updates of Netbox, skip the rpm and install according to documentation by cloning the git repo.

**Reply by TekkaTim** [reply]
Oh yeah, so it is. That's a shame.

I'll give the git cloning a go instead.

Ta

**Comment by DanSheps** [comment]
RPM on Fedora is not an approved method at all.  You should follow the installation instructions
